openquake/commands/tests/data/expected_converted_nrml.xml

```diff
@@ -0,0 +1,176 @@
+        <areaSource
+        id="1"
+        name="Quito"
+        tectonicRegion="Active Shallow Crust"
+        >
+            <areaGeometry>
+                <Polygon>
+                    <exterior>
+                        <LinearRing>
+                            <posList>
+                                -122.5 37.5 -121.5 37.5 -121.5 38.5 -122.5 38.5 -122.5 37.5
+                            </posList>
+                        </LinearRing>
+                    </exterior>
+                </Polygon>
+                <upperSeismoDepth>
+                    0.0
+                </upperSeismoDepth>
+                <lowerSeismoDepth>
+                    10.0
+                </lowerSeismoDepth>
+            </areaGeometry>
+            <magScaleRel>
+                PeerMSR
+            </magScaleRel>
+            <ruptAspectRatio>
+                1.5
+            </ruptAspectRatio>
+            <incrementalMFD binWidth="0.1" minMag="6.55" occurRates="0.0010614989 0.00088291627 0.00073437777 0.0006108288 0.0005080653"/>
+            <nodalPlaneDist>
+                <nodalPlane dip="90.0" probability="0.3" rake="0.0" strike="0.0"/>
+                <nodalPlane dip="45.0" probability="0.7" rake="90.0" strike="90.0"/>
+            </nodalPlaneDist>
+            <hypoDepthDist>
+                <hypoDepth hypodepth="4.0" probability="0.5"/>
+                <hypoDepth hypodepth="8.0" probability="0.5"/>
+            </hypoDepthDist>
+            <hypoList />
+            <slipList />
+        </areaSource>
+        <simpleFaultSource
+        id="3"
+        name="Mount Diablo Thrust"
+        tectonicRegion="Active Shallow Crust"
+        >
+            <simpleFaultGeometry>
+                <LineString>
+                    <posList>
+                        -121.823 37.73 -122.039 37.877
+                    </posList>
+                </LineString>
+            </simpleFaultGeometry>
+            <magScaleRel>
+                WC1994
+            </magScaleRel>
+            <ruptAspectRatio>
+                1.5
+            </ruptAspectRatio>
+            <incrementalMFD binWidth="0.1" minMag="5.0" occurRates="0.0010614989 0.00088291627 0.00073437777 0.0006108288 0.0005080653"/>
+            <nodalPlaneDist>
+                <nodalPlane dip="45.0" rake="30.0"/>
+            </nodalPlaneDist>
+            <hypoDepthDist />
+            <hypoList>
+                <hypo alongStrike="0.25" downDip="0.25" weight="0.3"/>
+                <hypo alongStrike="0.75" downDip="0.75" weight="0.7"/>
+            </hypoList>
+            <slipList>
+                <slip
+                weight="0.7"
+                >
+                    90.0
+                </slip>
+                <slip
+                weight="0.3"
+                >
+                    135.0
+                </slip>
+            </slipList>
+        </simpleFaultSource>
+        <NotImplemented id="5" name="characteristic source, simple fault" tectonicRegion="Volcanic"/>
```

Comment:
I am curious about the NotImplemented source, what is missing?

Reply:
In this case, the code of the missing ones is 'X'. Should we convert also them?

Reply:
Yes!

Reply:
For not implemented sources the following must happen:
1. they are not converted and skipped
2. in the log there is message (logging.error) telling which sources have been skipped